docs/technical-design/tealium-logging.md

```diff
@@ -0,0 +1,97 @@
+# Tealium Logging
+## Background
+We use a tool called Tealium to hook into the CMS customer analytics tools. The data is then sent to Adobe Analytics (previously CMS used Google Analytics). Access to analytics tools is set up through another team, Blast Analytics. They are contractors with a focus on analytics and work across all CMS domains. 
```

Comment:
Please link to appropriate monitoring.md section here, it explains what/why analytics in more detail

Reply:
I don't see a section for Tealium in `monitoring.md`. I think adding that section can be a follow-up PR for this so we can test this logging work out in production.